src/mlir/dialect/VeronaOps.td

```diff
@@ -183,6 +186,42 @@ def Verona_AllocateObjectOp : Verona_Op<"new_object"> {
     }];
 }
 
+def Verona_ReturnOp : Verona_Op<"return", [
```

Comment:
I'm still not convinced we need a special return. Sub-typing can be done as a pattern matching of cast+op.

Sure, it's easier for type checking if we bundle everything in one op, but it's harder for optimising and lowering if we have to treat standard operations as special.